src/server/kernel-mod/cplayground.c

```diff
@@ -203,6 +201,15 @@ static unsigned int get_containerized_processes(
     return container_tasks_len;
 }
 
+/**
+ * Gets the run state of a process, as is reported by ps and /proc/##/status.
+ * Returns the friendly single-character representation as opposed to the bitflag.
+ */
+static char get_proc_runstatus(struct task_struct *task) {
+    unsigned int friendly_state_id = task_state_index(task);
+	return task_index_to_char(friendly_state_id);
```

Comment:
Ah I don't have a C linter configured, but this should be unindented

Reply:
Yep, that's a typo.